aqandu/api_routes.py

```diff
@@ -290,7 +479,7 @@ def request_model_data_local(lat, lon, radius, start_date, end_date):
         )
         UNION ALL
         (
-            SELECT ID, time, PM2_5, Latitude, Longitude, '' as SensorModel, 'PurpleAir' as SensorSource
+            SELECT ID, time, PM2_5, Latitude, Longitude, "" as SensorModel, 'PurpleAir' as SensorSource
```

Comment:
The double quotes?  That must have been a typo on my part.   I have tested this and it works -- this was a refactoring so that we could have one call to make these queries.  If you think the double quotes are incorrect, please go ahead and fix them.